test.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import unittest
+
+
+class TestServerFunctionality(unittest.TestCase):
+
+	def setUp(self):
+		self.url= 'http://ec2-52-11-248-121.us-west-2.compute.amazonaws.com/api/v1'
+		self.loginval= {"login":"test", "password":"test"}
+
+	def test_1Login(self):
+		requestLogin = requests.post(self.url+'/login', json= self.loginval)
+		Token=(requestLogin.json().get('data')).get('tokenKey')
+		self.assertEqual(requestLogin.json().get('status'), 1)
+		EmailVerif= (requestLogin.json().get('data')).get('emailVerified')
+		self.assertEqual(EmailVerif,1)
+		userid= (requestLogin.json().get('data')).get('user_id')
+		self.assertEqual(EmailVerif,1)
+		return Token
+
+	def test_2Items(self):
+		Token= self.test_1Login()
```

Comment:
use class field for token
''''
+class TestServerFunctionality(unittest.TestCase):
     token = ""
''''